CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+`pwru` is an open source project. The userspace code is licensed under [Apache-2.0](LICENSE), while the BPF under [BSD 2-Clause](bpf/LICENSE.BSD-2-Clause) and [GPL-2.0](bpf/LICENSE.GPL-2.0). Everybody is welcome to contribute. Contributors are required to follow the [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) and must adhere to the [Developer Certificate of Origin](https://developercertificate.org/) by adding a Signed-off-by line to their commit messages.
+
+## Getting Started
+
+- Read the [README](https://github.com/cilium/pwru#readme) to understand how `pwru` works and its use cases.
+- Check out the [issues list](https://github.com/cilium/pwru/issues) for open tasks, especially those labeled `good first issue`.
+
+## Development Guidelines
+
+- Follow Cilium's general [Developer Guide](https://docs.cilium.io/en/stable/contributing/development/) for coding standards and best practices.
```

Comment:
The developer guide link here has a _lot_ of content, and it is all tailored towards developing on cilium/cilium. I'm not sure whether new contributors for pwru without cilium background would have a good chance of finding the relevant info here. I agree in principle the coding standards & best practices are likely well aligned between the subprojects, but I'm not sure which parts of the linked contributing guide are relevant for pwru.

Reply:
Something like https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#how-to-contribute is probably more practical, but there's still a few steps that aren't necessary for pwru, such as steps 7, 9-13.

Reply:
Overall the ideal is probably more that we enforce the "coding standards and best practices" via GitHub workflows so they apply automatically rather than just documenting them and hoping contributors will follow. But that would involve more work on the GHA side in this repo. I'm thinking stuff like adding golangci-lint to lint for common errors for instance.